src/legacy/core_plugins/embeddable_api/public/triggers/execute_trigger_actions.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Action, actionRegistry } from '../actions';
+import { openContextMenu } from '../context_menu_actions';
+import {
+  buildEuiContextMenuPanels,
+  ContextMenuAction,
+  ContextMenuPanel,
+} from '../context_menu_actions';
+import { IEmbeddable } from '../embeddables';
+import { trackUiMetric } from '../../../ui_metric/public';
```

Comment:
why are we using `trackUiMetric` ? it don't think we should add this in first PR, it will just make moving to NP harder. or atleast import it on top level only and then pass it down.

Reply:
removing, as discussed, will punt to a follow up PR/discussion